examples/animation/gltf_skinned_mesh.rs

```diff
@@ -0,0 +1,63 @@
+use std::f32::consts::PI;
+
+use bevy::{pbr::AmbientLight, prelude::*};
+
+/// Skinned mesh example with mesh and joints data loaded from a glTF file.
+/// Example taken from https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_019_SimpleSkin.md
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(AmbientLight {
+            brightness: 1.0,
+            ..Default::default()
+        })
+        .add_startup_system(setup.system())
+        .add_system(joint_animation.system())
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    // Create a camera
+    let mut camera = OrthographicCameraBundle::new_2d();
+    camera.orthographic_projection.near = -1.0;
+    camera.orthographic_projection.far = 1.0;
+    camera.orthographic_projection.scale = 0.005;
+    camera.transform = Transform::from_xyz(0.0, 1.0, 0.0);
+    commands.spawn_bundle(camera);
+
+    // Spawn the first scene in SimpleSkin.gltf
+    commands.spawn_scene(asset_server.load::<Scene, _>("models/SimpleSkin/SimpleSkin.gltf#Scene0"));
+}
+
+// The scene hierachy currently looks somewhat like this:
+//
+// <Parent node>
+// + Mesh node (without PBR bundle or SkinnedMesh component)
+//   + PBR entity (with PBR bundle and SkinnedMesh component, created by glTF loader)
+//   + First joint
+//     + Second joint
+//
+// In this example, we want to get and animate the second joint
+fn joint_animation(
+    time: Res<Time>,
+    parent_query: Query<&Parent, With<SkinnedMesh>>,
+    children_query: Query<&Children>,
+    mut transform_query: Query<&mut Transform>,
+) {
+    // Iter SkinnedMesh entity
+    for parent in parent_query.iter() {
+        // Get Children in the mesh node
+        let children = children_query.get(parent.0).unwrap();
+
+        // Get Children in the first joint
+        let children = children_query.get(children[1]).unwrap();
+
+        // Get Transform in the second joint
+        let mut transform = transform_query.get_mut(children[0]).unwrap();
```

Comment:
I'm having a bit of trouble following the [0] / [1] notation used here. Maybe break them out into seperate variables?